src/pages/preferences/preferencesTabs/NotificationsTab.tsx

```diff
@@ -135,20 +139,24 @@ const handleFieldUpdate = (stateUpdate) => (e) => {
   }
 };
 
+const StyledTextInput = styled(TextInput)`
+  margin-bottom: ${size.m};
+  width: 50%;
+`;
+
 const GridContainer = styled.div`
   display: grid;
-  grid-template-columns: 2fr repeat(3, 1fr);
+  grid-template-columns: repeat(2, 1fr);
   grid-template-rows: repeat(7, 1fr);
-  column-gap: 0;
-  row-gap: 0;
-  width: 50%;
+  margin-bottom: ${size.s};
+  width: 350px;
 `;
 
-const GridField = styled.div`
-  grid-area: ${(props: { gridArea: string }): string => props.gridArea};
-`;
+const NotificationMethod = styled.div`
```

Comment:
[nit] What do you think of including this in NotificationField, maybe by adding a `header` or `columns` prop to pass in the column labels? Colocating it with the content component feels like it may be better.

Reply:
Yes, that's a good point that it's confusing to maintain a grid across two files. I put all of the grid-related stuff into the same file!